src/components/accessSchedule/accessSchedule.js

```diff
@@ -51,7 +51,7 @@ import template from './accessSchedule.template.html';
         };
 
         if (parseFloat(updatedSchedule.StartHour) >= parseFloat(updatedSchedule.EndHour)) {
-            return void alert(globalize.translate('ErrorStartHourGreaterThanEnd'));
+            return alert(globalize.translate('ErrorStartHourGreaterThanEnd'));
```

Comment:
The `void` operator always returns `undefined`. In this cases, it would be equivalent to doing the following:
```
alert(globalize.translate('ErrorStartHourGreaterThanEnd'));
return;
```

I believe the void statements are just remnants from the original Emby source code being minified.

Reply:
Got it. Thanks.